rr_platform/launch/logitech_camera.launch

```diff
@@ -9,9 +9,6 @@
       <param name="height" value="90"/>
       <param name="video_mode" value="uncompressed"/>
       <param name="frame_rate" value="20"/>
```

Comment:
Need to recalibrate our cameras and replace the camera_info_url parameter we removed here.